.github/issue_template.md

```diff
@@ -0,0 +1,56 @@
+<!--- Provide a general summary of the issue in the Title above -->
+
+## I'm submitting a…
+
+<!-- Check one of the following options -->
+
+* [ ] Regression (a behavior that used to work and stopped working in a new release)
+* [ ] Bug report <!-- Please search GitHub for a similar issue or PR before submitting -->
+* [ ] Feature request
+* [ ] Documentation issue or request
+* [ ] Support request
+
+## Expected Behavior
+
+<!---
+  If you‘re describing a bug, tell us what should happen
+  If you‘re suggesting a change/improvement, tell us how it should work
+-->
+
+## Current Behavior
+
+<!---
+  If describing a bug, tell us what happens instead of the expected behavior
+  If suggesting a change/improvement, explain the change from current behavior
+-->
+
+## Possible Solution
+
+<!---
+  Not obligatory, but feel free to suggest a fix/reason for the bug,
+  or ideas how to implement the addition or change
+-->
+
+## Steps to Reproduce (for bugs)
+
+<!---
+  Provide a link to a live example or an unambiguous set of steps to
+  reproduce this bug. Include code, screenshot(s), or a screen recording
+  if relevant
+-->
+
+1. step 1
+2. step 2
+3. step 3
+4. step 4
+
+## Environment
+
+<!--- Include as many relevant details about the environment you experienced the bug in -->
+
+* caravan version: <!-- run `npm ls carvan` -->
+* unchained-bitcoin version <!-- run `npm ls unchained-bitcoin` -->
+* unchained-wallets version <!-- run `npm ls unchained-wallets` -->
+* Node version: <!-- run `node -v` -->
+* NPM version: <!-- run `npm -v` -->
+* Operating system: <!-- Mac, Linux, Windows -->
```

Comment:
Might want to include browser version as well as any relevant hardware versions (firmware and hardware).

Reply:
Good point, will add.